docs/api/migration-guides/qiskit-runtime-use-case.mdx

```diff
@@ -0,0 +1,267 @@
+---
+title: Common use cases
+description: Common use cases when migrating from backend.run to Qiskit Runtime primitives
+---
+
+# Common use cases
+This topic describes how to migrate programs with the most commonly used types of circuits.  For a full example, see [End-to-end examples.](qiskit-runtime-examples)
+
+## Basic circuits
+The only changes when running basic circuits are how you run the job and retrieve results.  
+
+<Tabs>
+  <TabItem value="Updated" label="Runtime primitives">
+```python
+from qiskit_ibm_runtime import QiskitRuntimeService, SamplerV2 as Sampler
+
+# Define the service.  This allows you to access IBM Quantum systems.
+service = QiskitRuntimeService()
+
+# Get a backend
+backend = service.least_busy(operational=True, simulator=False)
+
+job = sampler.run([isa_circuit])
+result = job.result()
+# Get results for the first (and only) PUB
+pub_result = result[0]
+
+```
+  </TabItem>
+
+  <TabItem value="Legacy" label="backend.run">
+``` python
+from qiskit import IBMQ
+
+# Define provider and backend
+provider = IBMQ.load_account()
+
+job = backend.run(isa_circuit)
+result = job.result()
+```
+  </TabItem>
+</Tabs>
+
+<span id="migrate-options"></span>
+## Specifying options
+
+All options that were available with `backend.run` are available in the Qiskit Runtime primitives, but they are specified differently.  For more information, see [Migrate options](qiskit-runtime-options) and [Advanced Qiskit Runtime options.](../../run/advanced-runtime-options)
+
+<Tabs>
+  <TabItem value="Updated" label="Runtime primitives">
+
+```python
+from qiskit_ibm_runtime import SamplerV2 as Sampler
+
+# You can use auto-complete to see and complete the options.
+sampler.options.default_shots = 1024
+```
+
+  </TabItem>
+
+  <TabItem value="Legacy" label="backend.run">
+```python
+from qiskit import IBMQ
+
+backend.run(circuit, shots=1024)
+```
+  </TabItem>
+</Tabs>
+
+
+## Dynamic circuits
+
+To migrate programs that run dynamic circuits, change the run call.
+
+<Tabs>
+  <TabItem value="primitives" label="Sampler">
+    ```python  
+from qiskit_ibm_runtime import SamplerV2 as Sampler
+
+sampler = Sampler(backend)
+job = sampler.run([dynamic_circ])
```

Comment:
Wouldn't this snippet require an additional bracket? I am not 100% sure: job = `sampler.run([(dynamic_circ)])`

Reply:
I ran it as-is using the code from the Repeat until success tutorial and it ran OK, so I guess it's alright this way, but maybe it's SUPPOSED to have an additional bracket.  I mess them up all the time :) @jyu00 ?

Reply:
It doesn't require the extra `()`. When we started writing docs for V2 primitives (i.e. before dev was done), they needed to be specified as a tuple (e.g. `run([(circ,)])`). That was no longer required at the time of the release, but many parts of the docs still had that syntax. There was a PR recently to remove the extra `()`, so hopefully everything is consistent now.